sklearn/utils/extmath.py

```diff
@@ -482,6 +482,9 @@ def randomized_svd(
     .. [3] An implementation of a randomized algorithm for principal component
       analysis A. Szlam et al. 2014
 
+    See:ref:`sphx_glr_auto_examples_applications_wikipedia_principal_eigenvector.py`
```

Comment:
Move this change in the long summary of the function.

Instead of "how to use the API", basically the example shows a typical case where randomized SVD was used: ranking of web pages also known as PageRank.